packages/components/scripts/generateDSRComponents.ts

```diff
@@ -339,7 +338,7 @@ $&`
           );
       }
 
-      return newFileContent;
+      return '// @ts-nocheck\n' + newFileContent;
```

Comment:
That's not good. We don't want to disable every type checking in those generated files.  
Instead, the `implementes XY` part should be removed since the suggested change tries to solve that, right?